data/playground/python/values.py

```diff
@@ -37,9 +37,7 @@ def my_funk(value: str = "hello", other: bool=False) -> str:
 
 # we can say "change every value" to allow modifying all the values in one go
 def foo():
-    a = 0
-    b = 1
-    c = 2
+    pass

```

Comment:
wow looks like we have some pretty aggressive "auto-formatting" going on here 😅. cc/ @auscompgeek any idea which formatter is doing this?

Reply:
It's probably ruff's pyflakes (F) autofixes yeeting the unused variables, then the unused literal expressions?

Reply:
I guess I would prefer a blanket exception for everything in the playground. I have several things in the playground that are not gofmt compliant (but are valid go code ) specifically so that I can experiment to see whether cursorless does the right thing with them.

Reply:
I am very much on the fence, but lean slightly towards formatting playground but being liberal with comments to disable auto-formatter / linter where we want. @josharian is it not easy to disable gofmt on a per-line basis?

I don't feel strongly, tho. I could see the benefit to having full freedom in the playground

Reply:
> is it not easy to disable gofmt on a per-line basis

no, by design gofmt is impossible to selectively disable and has no knobs whatsoever

(Happy to expound upon the thinking behind those design decisions over voice, if desired)

Reply:
However, there are also no pre-commit hooks for Go at all, so far this is not a problem in the playground or anywhere. :)

Reply:
I could see a case for blanket disabling formatter in playground. That being said, the formatter has detected real syntax errors in these new python playground files, so it is providing some value here

Reply:
We could have the pre-commit step check for the actual property we want, namely valid syntax

Reply:
It depends in part on how active the formatter is. There is some danger of writing queries that only match when using the same formatter as us, particularly if the formatter is not whitespace only. 🤷🏽

Reply:
Yeah that's a good point. I think maximal flexibility is not necessarily a bad thing in the playground. Interesting idea to just check for valid syntax. Altho tbh we even might want to violate that in some cases because error tolerance is one of the reasons we like tree-sitter

Reply:
I'm convinced. See 43939f76e080c3000e9d23df27b1004fe667f359. Any objections @auscompgeek @josharian @AndreasArvidsson ?

Reply:
No sounds resonable